### PR TITLE
Update Pipfile packages

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
This PR removes `snakeviz` from our dev packages as the README instructs installation via Pip.
This also means all outdated Pipfile packages have been upgraded. 

*Pylint* upgrade identifies these new issues which has been resolved.
```bash
************* Module app.new_relic
app/new_relic.py:2:4: C0415: Import outside toplevel (newrelic.agent) (import-outside-toplevel)
************* Module app.globals
app/globals.py:14:4: C0415: Import outside toplevel (app.storage.encrypted_questionnaire_storage) (import-outside-toplevel)
app/globals.py:31:4: C0415: Import outside toplevel (app.data_model.session_store) (import-outside-toplevel)
app/globals.py:67:4: C0415: Import outside toplevel (app.data_model.session_store) (import-outside-toplevel)
************* Module app.questionnaire.questionnaire_store_updater
app/questionnaire/questionnaire_store_updater.py:88:0: R1721: Unnecessary use of a comprehension (unnecessary-comprehension)
************* Module app.setup
app/setup.py:352:8: C0415: Import outside toplevel (flask_debugtoolbar) (import-outside-toplevel)
app/setup.py:361:4: C0415: Import outside toplevel (app.routes.questionnaire) (import-outside-toplevel)
app/setup.py:366:4: C0415: Import outside toplevel (app.routes.questionnaire) (import-outside-toplevel)
app/setup.py:371:4: C0415: Import outside toplevel (app.routes.session) (import-outside-toplevel)
app/setup.py:377:4: C0415: Import outside toplevel (app.routes.flush) (import-outside-toplevel)
app/setup.py:383:4: C0415: Import outside toplevel (app.routes.dump) (import-outside-toplevel)
app/setup.py:388:4: C0415: Import outside toplevel (app.routes.errors) (import-outside-toplevel)
app/setup.py:393:4: C0415: Import outside toplevel (app.jinja_filters) (import-outside-toplevel)
app/setup.py:397:4: C0415: Import outside toplevel (app.routes.static) (import-outside-toplevel)
app/setup.py:402:4: C0415: Import outside toplevel (app.routes.schema) (import-outside-toplevel)
************* Module tests.app.questionnaire.test_pointers
tests/app/questionnaire/test_pointers.py:11:0: R1721: Unnecessary use of a comprehension (unnecessary-comprehension)
************* Module tests.app.storage.test_dynamodb
tests/app/storage/test_dynamodb.py:24:16: E1101: Instance of '' has no 'create_table' member (no-member)
************* Module tests.app.questionnaire.test_placeholder_renderer
tests/app/questionnaire/test_placeholder_renderer.py:85:0: R1721: Unnecessary use of a comprehension (unnecessary-comprehension)
************* Module tests.app.authentication.test_roles
tests/app/authentication/test_roles.py:28:8: C0415: Import outside toplevel (app.authentication.roles) (import-outside-toplevel)
************* Module tests.integration.integration_test_case
tests/integration/integration_test_case.py:55:8: C0415: Import outside toplevel (application) (import-outside-toplevel)
************* Module scripts.extract_translation_templates
scripts/extract_translation_templates.py:54:4: W1510: Using subprocess.run without explicitly set `check` is not recommended. (subprocess-run-check)

-------------------------------------------------------------------
Your code has been rated at 9.98/10 (previous run: 10.00/10, -0.02)

Pylint linting check failed
```
### How to review 
Ensure:
- App behaviour is unchanged.

### CODE REVIEW

**Comment by MebinAbraham** [comment]
[Update Pip packages](https://trello.com/c/E4cnNYxi/3502-update-pip-packages)

File: app/setup.py [hunk 1]
```diff
@@ -357,11 +357,14 @@ def start_dev_mode(application):
         application.config['DEBUG_TB_PROFILER_ENABLED'] = True
         application.config['DEBUG_TB_INTERCEPT_REDIRECTS'] = False
         application.debug = True
-        from flask_debugtoolbar import DebugToolbarExtension
+        from flask_debugtoolbar import (  # pylint: disable=import-outside-toplevel
+            DebugToolbarExtension,
+        )
```
Comment (ajmaddaford) [inline app/setup.py:362]: Does anyone use this? Could be a good time to remove it.

**Review by ajmaddaford — APPROVED** [review]

**Review by JamesGardiner — APPROVED** [review]
Approved based on a new card being added to track fixing suppressed pylint errors for imports not at the top of the module

File: app/setup.py (hunk 1 above)
Comment (MebinAbraham) [reply app/setup.py:362]: Have made a card to review this.

**Pull request merged**